subsys/net/lib/sockets/getaddrinfo.c

```diff
@@ -51,6 +51,7 @@ struct getaddrinfo_state {
 	int status;
 	uint16_t idx;
 	uint16_t port;
+	uint16_t dns_id;
```

Comment:
Should this value be set to a default value in `z_impl_z_zsock_getaddrinfo_internal`?

Reply:
I can set this to 0, I see that's what the resolver module does in case of failure.